diff --git a/node_modules/vite/src/node/plugins/css.ts b/node_modules/vite/src/node/plugins/css.ts
index dfcb70c..f6d4af2 100644
--- a/node_modules/vite/src/node/plugins/css.ts
+++ b/node_modules/vite/src/node/plugins/css.ts
@@ -906,7 +906,7 @@ async function minifyCSS(css: string, config: ResolvedConfig) {
   if (warnings.length) {
     const msgs = await formatMessages(warnings, { kind: 'warning' })
     config.logger.warn(
-      chalk.yellow(`warnings when minifying css:\n${msgs.join('\n')}`)
+      chalk.yellow(`${msgs.length} warnings when minifying css`)
     )
   }
   return code
